docs/3.commands_and_usage.md

```diff
@@ -153,6 +160,56 @@ results could not be compared in 0 places:
 
 If only one file is provided to the `compare` command, then the hashes in the file are compared with hashes of the current state of the working directory. In that case, it is possible to also specify paths to the `input_data`, `code` and `output_data` (otherwise the default values are used).
 
+### config
+
+The other commands `engage`, `disengage` and `compare` use a common set of arguments:
+
+Required `--input_data`, `--code`, `--output_data`.
+Optional `--csv`, `--catalogue_results` (see below)
+
+The `config` command creates a configuration file with values for the above arguments.
+This allows you to specify the arguments just once and have them used for each of
+`engage`, `disengage` and `compare`. It is recommended you run this command first.
+
+```bash
+catalogue config --input_data <data directory> --code <code directory> --output_data <results directory> --csv <csv_file_name> --catalogue_results <versioning_files>
+```
+
+This will create a `catalogue_config.yaml` configuration file in the root repository with the following format:
+
+```
+input_data: <data directory>
+code: <code directory>
+output_data: <results directory>
+csv: <csv_file_name>
+catalogue_results: <versioning_files>
+```
+
+We can now run commands without specifying the full arguments. The arguments will
+instead be taken from the configuration file. Repro-catalogue uses the following priority ordering for arguments:
+
+specified arguments > configuration arguments > default arguments
+
+For example running:
+
+``bash
+catalogue --engage <data directory2>
+``
+
+Is equivalent to running:
+
+``bash
+catalogue --engage --input_data <data directory2> --code <code directory>
+``
+
+Here we have specified an exact argument so it has been selected. No arguments however have been chosen for `--code`
+and so the parser instead looks inside the `catalogue_config.yaml` file to find the `<code directory` argument value.
+
+We can also manually edit the `catalogue_config.yaml` configuration file, but it needs to retain a similar format
```

Comment:
Instead of "a similar format" I think you should be more precise and say "the illustrated `keyword: value` format"